jekyll/_cci2/runner-installation.adoc

```diff
@@ -0,0 +1,154 @@
+
+This will print a generated Runner config including the authentication token. Note that the token cannot be retrieved again, so store it safely.
+
+==== Installation Tooling
+
+The installation process assumes that the following utilities are installed on the system:
+
+- curl (installed by default on macOS)
+- sha256sum (installed as part of coreutils on Linux apt/yum, macOS via brew)
+- systemd version 235+ (Linux only)
+- Installation also requires permissions to create a user, and create directories under /opt.
+
+=== Job Running Requirements
+
+Running jobs requires that the following tools are available on the machine:
+
+- tar
+- gzip
+- coreutils (Linux only)
+- git (recommended, but not required)
+
+=== Installation
+
+==== Download the Launch Agent Binary and Verify the Checksum
+
+The Launch Agent can be installed using the following script in a bash shell.
+
+This will use /opt/circleci as the base install location.
+
+First, set one of these variables as appropriate for for your installation target.
+
+===== For Linux x86_64
+
+`platform=linux/amd64`
+
+===== For Linux ARM64
+
+`platform=linux/arm64`
+
+===== For macOS x86_64
+
+`platform=darwin/amd64`
+
+And then run the following steps to download, verify and install the binary.
+
+```sh
+prefix=/opt/circleci
+sudo mkdir -p "$prefix/workdir"
+base_url="https://circleci-binary-releases.s3.amazonaws.com/circleci-launch-agent"
+echo "Determining latest version of CircleCI Launch Agent"
+agent_version=$(curl "$base_url/release.txt")
+echo "Using CircleCI Launch Agent version $agent_version"
+echo "Downloading and verifying CircleCI Launch Agent Binary"
+curl -sSL "$base_url/$agent_version/checksums.txt" -o checksums.txt
+file="$(grep -F "$platform" checksums.txt | cut -d ' ' -f 2)"
+file="${file:1}"
+mkdir -p "$platform"
+echo "Downloading CircleCI Launch Agent: $file"
+curl --compressed -L "$base_url/$agent_version/$file" -o "$file"
+echo "Verifying CircleCI Launch Agent download"
+sha256sum --check --ignore-missing checksums.txt && chmod +x "$file"; sudo cp "$file" "$prefix/circleci-launch-agent" || echo "Invalid checksum for CircleCI Launch Agent, please try download again"
+```
+
+=== Platform-specific instructions
+
+Please refer to the platform-specific installation instructions:
+
+- linux
+- macOS
+
+=== Configuration file reference
+
+A YAML file is used to configure the Launch Agent; how it communicates with our servers and how it will launch the Task Agent.
+
+The configuration file uses the following format, the parameters are explained in more detail below:
+
+```sh
+api:
+  auth_token: AUTH_TOKEN
+runner:
+  name: RUNNER_NAME
+runner.name
+```
+
+This is a unique name assigned to this particular running Launch Agent, we recommend using the hostname of the machine. It will be used to identify the agent when viewing statuses and job results in the CircleCI UI.
```

Comment:
I'm not sure it's clear what "This" is referring to. Should this sentence start with `RUNNER_NAME` is a unique name...

Reply:
or rather, perhaps this is missing the `==== runner.name` heading.

Reply:
fixed